python/graphstorm/sagemaker/sagemaker_partition.py

```diff
@@ -386,6 +392,75 @@ def data_dispatch_step(partition_dir):
 
         data_dispatch_step(local_partition_path)
 
+        # Leader instance copies raw_id_mappings from input to dist_graph output on S3
+        # using aws cli, usually much faster than using boto3
+        # Will also upload the updated GConstruct/GSProcessing config, if one exists
+        raw_id_mappings_s3_path = os.path.join(graph_data_s3, "raw_id_mappings")
+        logging.info("Copying raw_id_mappings from %s to %s",
+            raw_id_mappings_s3_path, s3_dglgraph_output)
+        # Copy raw_id_mappings from input to dist_graph output on S3
+        subprocess.call([
+            "aws", "configure", "set", "default.s3.max_concurrent_requests", "150"])
```

Comment:
How do we get 150 here? Is this value instance dependent?

Reply:
Around 100 is a good value when looking for parallel download capacity. We can select a safer value but for all instances I've used this is an OK value. The max value will depend on a number of factors but 50-150 should be safe for most instance types